jquery.xblock.js

```diff
@@ -214,7 +221,7 @@
 
                         xhr.setRequestHeader('X-Requested-With', 'XMLHttpRequest');
 
-                        if (!$this.csrfSafeMethod(settings.type) && queryDomain === lmsDomain) {
+                        if (!$this.csrfSafeMethod(settings.type)) {
```

Comment:
@Kelketek not sure which one is right here. @antoviaque what was the result of the conversation about this?